content/en/docs/release-oversight/testing-in-production.md

```diff
@@ -0,0 +1,57 @@
+
+* Did we have a build on time?
+* Were any changes required by the administrator missing an [admin ack](https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare)?
+* Did we find a significant problem that forced us to remove the production cluster from rotation?
+
+### Selecting the build
+The selection process is quite simple.  On the last Wednesday of each sprint the [Technical Release Team](/docs/release-oversight/the-technical-release-team/) (TRT) will select an accepted payload from the last two days.  If no payload is readily available and time allows for deeper investigation, the TRT may manually accept a payload.  This involves carefully accounting for extraneous circumstances that can lead to widespread payload blockage.  Cloud provider or other infrastructure related problems are just two examples.
+
+From there Quality Engineering will run additional tests.  Barring any new surprises Test Platform will be notified that a release is ready.
+
+It's worth noting that we may create a dedicated channel for managing these releases through the [Update Service](https://docs.openshift.com/container-platform/4.7/updating/understanding-the-update-service.html) in the upcoming months.  At the present we classify the sprint payload as an *fc* build which notifies Quality Engineering that additional testing is needed.
+
+### The environment
+We've selected one of our build farm clusters to run the output of each sprint.  This is a critical piece of infrastructure that all development teams are motivated to keep working.  In this document we'll refer to this cluster as the *sacrificial cluster*.
+
+### Investigating problems
+The TRT has admin access to the sacrificial cluster for debugging purposes.  As needed this access can be temporarily granted by Test Platform to any other development team that needs it.  Temporary configuration changes are allowed for experiments but anything needed to persist will require failing the sprint.
+
+### Hotfixes
+Hotfixes are possible but not desired.  Changes for the sole purpose of gathering more data are preferred over changes in functionality.  Before we resort to patching a problem we'll default to [reverting the change](/docs/release-oversight/improving-ci-signal/).  Changes like this are best done as part of the subsequent sprint release.
+
+### When things go wrong the sacrificial cluster is removed from rotation
+The Test Platform team maintains the infrastructure that powers our CI builds.  When there are major outages they already have an incident management process in place.  What's different now is that the TRT can detect subtle degradations as a result of sprint upgrades.  In most of these cases the TRT contacts Test Platform via their helpdesk bot to request the sacrificial cluster be removed from rotation.  Behind the scenes a Jira is created for tracking.
```

Comment:
```suggestion
The Test Platform team maintains the infrastructure that powers our CI builds.  When there are major outages they already have an incident management process in place.  What's different now is that the TRT can detect subtle degradations as a result of sprint upgrades.  In most of these cases the TRT contacts Test Platform team via their helpdesk bot to request the sacrificial cluster be removed from rotation.  Behind the scenes a Jira is created for tracking.
```

Reply:
I think it's OK to refer to the team casually as "Test Platform".  I have at least 10 other references where I do so.  I may very well take your suggestion the next time I'm making updates to this doc though.